packages/core/examples/video/index.ts

```diff
@@ -100,13 +101,30 @@ addButtonToToolbar({
   },
 });
 
+const useLocal = true;
```

Comment:
can you remove this pleaes?

Reply:
No, it is important to be able to test against local easily.  I will set it to false by default  - wasn't supposed to get committed true, but the basic ability is important for consistent testing.